FreeRTOS-Plus/Demo/AWS/Fleet_Provisioning_Windows_Simulator/Fleet_Provisioning_With_CSR_Demo/fleet_provisioning_demo_setup.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
```

Comment:
Is there a requirements.txt for this python script?

Consider using type hints and a linter like mypy to enforce these.

Reply:
There is no requirements.txt for the python script, the only package dependency is cryptography (which will be noted in the documentation, which details how to use the Python script).